README.md

```diff
@@ -11,37 +12,41 @@
 - [Desktop app](#desktop-app)
 - [Sistema](#sistema)
 
-----
+---
+
 ## Before Installation
 
 First you need to have NVM, NODE and YARN installed.
 
 To install [nvm](https://github.com/nvm-sh/nvm#installing-and-updating), run:
+
 ```
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
 ```
+
 or
+
 ```
 wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
 ```
 
-To check if it was installed properly, close and reopen the terminal and run "```command -v mvn```" and should return "```mvn```". In case there is something else going on, troubleshoot with [this documentation](https://github.com/nvm-sh/nvm#troubleshooting-on-macos). To see all the commands, simply run "```nvm```".
-
+To check if it was installed properly, close and reopen the terminal and run "`command -v mvn`" and should return "`mvn`". In case there is something else going on, troubleshoot with [this documentation](https://github.com/nvm-sh/nvm#troubleshooting-on-macos). To see all the commands, simply run "`nvm`".
```

Comment:
```suggestion
To check if it was installed properly, close and reopen the terminal and run "`command -v nvm`" and should return "`nvm`". In case there is something else going on, troubleshoot with [this documentation](https://github.com/nvm-sh/nvm#troubleshooting-on-macos). To see all the commands, simply run "`nvm`".
```